lms/static/js/learner_dashboard/models/course_card_model.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Model for Course Programs.
+ */
+(function (define) {
+    'use strict';
+    define([
+            'backbone'
+        ], 
+        function (Backbone) {
+        return Backbone.Model.extend({
+            initialize: function(data) {
+                if (data){
+                    this.context = data;
+                    //we should populate our model by looking at the run_modes
+                    if (data.run_modes.length > 0){
+                        //We only have 1 run mode for this program
+                        this.setActiveRunMode(data.run_modes[0]);
```

Comment:
@schenedx we're going to need to do a little more than this to determine the active run-mode. Picking the active run-mode requires either 1) showing the latest run-mode in which the user is enrolled, if any, or 2) showing the first run-mode in which enrollment is open.